build_tools/third_party/tensorflow/tensorflow/compiler/mlir/hlo/CMakeLists.txt

```diff
@@ -186,9 +187,9 @@ external_tablegen_library(
   ROOT
     ${TF_MLIR_HLO_SRC_ROOT}
   SRCS
-    "lib/Dialect/mhlo/transforms/canonicalize.td"
+  "lib/Dialect/mhlo/IR/mhlo_canonicalize.td"
```

Comment:
NIT: "Missing" indent?

Reply:
Oops, sorry missed this before clicking merging internally..

I guess this is gonna change very soon again with MHLO separating out from TensorFlow repo. So I won't bother sending another CL to fix it; it can be coupled with the next round of changes. :)